main.py

```diff
@@ -14,6 +14,7 @@
 parser.add_argument('-v', '--verbose', action="store_true", help="Enable verbose output")
 parser.add_argument('--output', action="store", default=None, help="Output file")
 parser.add_argument('--as-index', action="store_true", default=False, required=False, help="Outputs sitemap as index and multiple sitemap files if crawl results in more than 50,000 links (uses filename in --output as name of index file)")
+parser.add_argument('--no-sort',  action="store_false", default=True, required=False, help="Disables sorting the output URLs alphabetically", dest='sort_alphabetically')
```

Comment:
If you enable this arg, how does the rest of the code know about that?

Reply:
The flag is `--no-sort` but the actual value is returned in `sort_alphabetically` (see at the end we have `dest=sort_alphabetically`)
I just wanted to avoid using a variable to store a negation of a condition, especially since we are interested in the actual condition. Was trying to avoid:
```
if not self.no_sort:
   // sort
```
Because that made it more difficult to read in my opinion, as opposed to 
```
if self.sort:
   // sort
```
But, if you think I'm making it more complicated than it should be, we can do it the opposite way

Reply:
Ah got it, thanks!